packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -31,7 +31,7 @@ export const getAssetUrl = (path: string) => {
     const publicPath = onClient
         ? // @ts-ignore
           `${window.Progressive.buildOrigin as string}`
-        : `${bundleBasePath as string}/${process.env.BUNDLE_ID || 'development'}/`
+        : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}/`
```

Comment:
Both my local and ci servers failed on this line. I'm not sure why it was passing before, it's possible the linting package was updated and this error popped up, but I've run lint fix and fixed the issue. 

Just wanted you to be aware of this change if anything pops up on your end. @vmarta